io.openems.edge.evcs.webasto.next/src/io/openems/edge/evcs/webasto/next/EvcsWebastoNextImpl.java

```diff
@@ -0,0 +1,322 @@
+				new FC3ReadRegistersTask(1028, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.POWER_L2, new UnsignedDoublewordElement(1028))),
+				new FC3ReadRegistersTask(1032, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.POWER_L3, new UnsignedDoublewordElement(1032))),
+				new FC3ReadRegistersTask(1036, Priority.LOW,
+						m(Evcs.ChannelId.ACTIVE_CONSUMPTION_ENERGY, new UnsignedDoublewordElement(1036))),
+				new FC3ReadRegistersTask(1100, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.MAX_HW_CURRENT, new UnsignedWordElement(1100))),
+				new FC3ReadRegistersTask(1102, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.MIN_HW_CURRENT, new UnsignedWordElement(1102))),
+				new FC3ReadRegistersTask(1104, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.MAX_EVSE_CURRENT, new UnsignedWordElement(1104))),
+				new FC3ReadRegistersTask(1106, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.MAX_CABLE_CURRENT, new UnsignedWordElement(1106))),
+				new FC3ReadRegistersTask(1108, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.MAX_EV_CURRENT, new UnsignedWordElement(1108))),
+				// TODO EvcsWebastoNext.ChannelId.LAST_ENERGY_SESSION: This register remains 0
+				// during the session,
+				// and set a value at the end. But for the UI we need the
+				// Energy charged during the session is running as well.
+				new FC3ReadRegistersTask(1502, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.LAST_ENERGY_SESSION, new UnsignedWordElement(1502))),
+				new FC3ReadRegistersTask(1504, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.START_TIME, new UnsignedDoublewordElement(1504))),
+				new FC3ReadRegistersTask(1508, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.CHARGE_SESSION_TIME, new UnsignedDoublewordElement(1508))),
+				new FC3ReadRegistersTask(1512, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.END_TIME, new UnsignedDoublewordElement(1512))),
+				new FC3ReadRegistersTask(1620, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.SMART_VEHICLE_DETECTED, new UnsignedWordElement(1620))),
+				new FC3ReadRegistersTask(2000, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.SAFE_CURRENT, new UnsignedWordElement(2000))),
+				new FC3ReadRegistersTask(2002, Priority.LOW,
+						m(EvcsWebastoNext.ChannelId.COM_TIMEOUT, new UnsignedWordElement(2002))),
+				new FC16WriteRegistersTask(5000, //
+						m(EvcsWebastoNext.ChannelId.EV_SET_CHARGE_POWER_LIMIT, new UnsignedDoublewordElement(5000))), //
+				new FC6WriteRegisterTask(5004, //
+						m(EvcsWebastoNext.ChannelId.CHARGE_CURRENT, new UnsignedWordElement(5004))), //
+				new FC6WriteRegisterTask(5006, //
+						m(EvcsWebastoNext.ChannelId.START_CANCEL_CHARGING_SESSION, new UnsignedWordElement(5006))),
+				new FC3ReadRegistersTask(6000, Priority.LOW, //
+						m(EvcsWebastoNext.ChannelId.LIFE_BIT, new UnsignedWordElement(6000))), //
+				new FC6WriteRegisterTask(6000, //
+						m(EvcsWebastoNext.ChannelId.LIFE_BIT, new UnsignedWordElement(6000))) //
+		);
+		this.addStatusListener();
```

Comment:
Addendum: A listener for the currently used phases is missing, as long as the CurrentL1/L2/L3 values are not part of the nature and the logic is not handled by them

Reply:
I have added the listener. Thanks for the hint.